src/shadowbox/server/api.yml

```diff
@@ -27,7 +27,34 @@ paths:
               examples:
                 '0':
                   value: >-
-                    {"name":"My Server","serverId":"000-000-000-000","metricsEnabled":true,"createdTimestampMs":1536613192052,"portForNewAccessKeys":1234}
+                    {"name":"My Server","serverId":"000-000-000-000","metricsEnabled":true,"createdTimestampMs":1536613192052,"portForNewAccessKeys":1234, "version": "1.0.0"}
+  /server/hostname:
+    put:
+      tags:
+        - Server
+      description: Changes the server's hostname.  Must be a valid hostname or IP address.
```

Comment:
We should clarify this description. Maybe say something like "Change the host to be returned in access key URLs. Must be a domain name or IP address"

Reply:
Done, and added a note about DNS